contrib/harfbuzz/module.defs

```diff
@@ -18,11 +14,7 @@ HARFBUZZ.CONFIGURE.extra = \
     FREETYPE_CFLAGS="-I$(call fn.ABSOLUTE,$(CONTRIB.build/))include/freetype2" \
 
 ifneq ($(BUILD.system),darwin)
```

Comment:
OK, I went with your original suggestion for now, as it seems the code is a little clearer this way: a separate block for FreeType, then a separate block for Fontconfig. If you’d rather have them merged after all (with an updated comment), let me know!